assets/javascripts/discourse/initializers/chat-sidebar.js

```diff
@@ -249,12 +250,21 @@ export default {
               return this.channel.title;
             }
 
+            get oneOnOneMessage() {
+              return this.channel.chatable.users.length === 1;
+            }
+
             get text() {
-              return this.channel.title.replaceAll("@", "");
+              const username = this.channel.title.replaceAll("@", "");
+              if (this.oneOnOneMessage) {
+                return htmlSafe(`${username} ${decorateUsername(username)}`);
```

Comment:
Should `username` be here twice? Or does `decorateUsername(username)` not include the `username` in its output?

Reply:
Yes, decorateUsername only contains suffix flairs. 

Example usage of decorate-username-selector is below. We display username and then suffix
https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/templates/user-selector-autocomplete.hbr#L7;L9